mui/mui.py

```diff
@@ -149,10 +150,37 @@ def solve(bv: BinaryView):
         )
 
         if dialog.exec() == QDialog.Accepted:
+            if notif.mui_grpc_server_process == None:
+                notif.mui_grpc_server_process = subprocess.Popen(
+                    [Path(__file__).resolve().parent.parent / "muicore_server"]
```

Comment:
I don't think this is safe for when ManticoreUI is installed through Binary Ninja's plugin installer.

Since Binary Ninja is pure Python, we should have the grpc server exist as a dependency and be built on demand when someone installs ManticoreUI.

We can punt on actually doing this until we have more of the essential features implemented, but it's something we need to track (or at least remember to test usage of ManticoreUI as installed through the plugin manager)